docs/installation/upgrade/include/2.4-removed-settings.inc

```diff
@@ -0,0 +1,44 @@
+#### Settings
+
+| Setting | Notes |
+| ------- | ----- |
+| `auth_debug` | Use [[setting,log_debug]] filter instead. Example: `log_debug=category=auth`. |
+| `auth_default_realm` | Replaced by [[setting,auth_default_domain]]. |
+| `auth_stats` | |
+| `auth_worker_max_count` | Use [[link,service_configuration,service-specific process limit]]. |
+| `dict_db_config` | Berkeley DB is not supported anymore. |
+| `disable_plaintext_auth` | Replaced by [[setting,auth_allow_cleartext]]. |
+| `imap_id_log` | Replaced by the [[event,imap_id_received]] event. |
+| `login_access_sockets` | Use [[link,auth_lua]] instead. Dovecot will fail to start if this setting is present in configuration. |
+| `push_notification_backend` | Use [[setting,push_notification_driver]] instead. |
+| `quota_set` | |
+| `sieve_dir` | See [[link,sieve_file]] and [[link,sieve_location]]. |
+| `sieve_global_dir` | See [[link,sieve_file]] and [[link,sieve_location]]. |
+| `sieve_global_path` | See [[link,sieve_file]] and [[link,sieve_location]]. |
+| `sieve_vacation_max_subject_codepoints` | |
+| `auth_policy_server_timeout_msecs` | See: [[link,auth_policy_configuration]]. |
+| `mail_crypt_require_encrypted_user_key` | Replaced by [[setting,crypt_user_key_require_encrypted]], [[setting,crypt_user_key_password]] and [[setting,crypt_user_key_encryption_key]]. |
+| `verbose_ssl` | Use [[setting,log_debug,category=ssl]] instead. |
+| `mail_location` setting & `mail` userdb field | Split into multiple [[link,mail_location,mail_*]] settings. |
+| `listescape` plugin | [[setting,mailbox_list_storage_escape_char]] setting. |
```

Comment:
Should be in 2.4-removed-plugins.inc

Reply:
Although current 2.4-removed-plugins.inc is only v3.0 stuff, which is included in v3.0 docs. If you add this there, it's only in v3.1. Do we need 2.4-removed-plugins.inc & 2.4-removed-plugins-configrewrite.inc or something?

Reply:
we can include 2.4-includes in 3.0 docs.